mkdocs_drawio_exporter/exporter.py

```diff
@@ -289,9 +317,12 @@ def replace(match):
                         self.log.error(f'Export failed with exit status {exit_status}; skipping rewrite')
                         return match.group(0)
 
-                    with open(img_path, "r") as f:
+                    with open(img_path, "r", encoding="utf-8") as f:
                         content = f.read()
 
+                    if config["link_css"] or re.findall(r'link_css="(?:true|yes|1)"', match.group(0)):
```

Comment:
I wonder if we should invert this condition to check for falsy values and assume other values are truthy?

```suggestion
                    if config["use_theme_colors"] or not re.findall(r'use_theme_colors="(?:false|no|0)"', match.group(0)):
```